content/rc/administration/configuration/_index.md

```diff
@@ -7,7 +7,7 @@ categories: ["RC"]
 aliases: /rc/administration/configure/
         /rv/administration/configuration/
 ---
-While there is very little configuration of Redis Cloud
+While there is little configuration of Redis Cloud
```

Comment:
```suggestion
Although Redis Cloud requires very little configuration,```

Reply:
I suggest keeping "very" in this case, but rewording.

Reply:
Updated.